docker-compose.yaml

```diff
@@ -6,12 +6,20 @@ services:
     expose:
       - 8000
     secrets:
-      - mendeley-secret
+      - asc-secret
     stop_signal: SIGINT
     restart: always
 
-  redis:
-    image: redis:alpine
+  mongo:
+    image: mongo:4.2-bionic
+    restart: always
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: root
+      MONGO_INITDB_ROOT_PASSWORD: integration
+    ports:
+      - 27017:27017
```

Comment:
this should probably be `- 127.0.0.1:27017:27017` otherwise this would become globally bound

Reply:
Yes very good point!